resources/todomvc/architecture-examples/svelte/dist/index.html

```diff
@@ -18,4 +18,4 @@
         </footer>
         <script src="app.js"></script>
     </body>
-</html>
+</html>
```

Comment:
Mmm on my local machine, when I run `npm run build` in svelte, I get the previous file instead.
What do you see on your side?

Reply:
Maybe I need more coffee - what's exactly the change here?
I do see a changed index.html file as well...

Reply:
A new line character has been added or removed (I don't remember which one :) )

Reply:
Fixed.